designs/administrator/future-openshift/resource-related-quick-starts/index.md

```diff
@@ -0,0 +1,65 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Surfacing resource-related quick starts
+
+## Introduction
+Sometimes quick starts can be closely associated with a resource type, for example, a virtual machine or a pipeline. With that in mind, it became important to surface quick starts on that resource's page to improve the flow of both creating a resource and using a resource. This design demonstrates how to do so.
+
+## Design summary
+The design is broken up into scenarios:
+- [Populated resource list views](#populated-resource-list-views)
+- [Empty resource list views](#empty-resource-list-views)
+
+Whether the list view is populated or not, the general pattern of this design includes surfacing a dismissible label to make it clear that a quick start is available within the page heading.
+![general-design](img/general-design.png)
+
+The label is more like a hint and can be hidden, which would then hide **all** resource-related quick start labels for that user.
```

Comment:
I am wondering if dismissing this resource-related label should also hide the ones on the operator details page. Was there consideration to not making these dismissible? Or maybe that is overly forceful? They do seem somewhat lightweight on the page to maybe just allow them to always be present.

Reply:
There was consideration in the beginning to make them non-dismissible, but then a lot of feedback came rolling in that they should be dismissible once I shared at feedback sessions